README.md

```diff
@@ -74,6 +74,8 @@ To learn the basic commands and enable professional Ubuntu services in a sandbox
 * [What are the timer jobs?](./docs/explanations/what_are_the_timer_jobs.md)
 * [What are the UA related MOTD messages?](./docs/explanations/motd_messages.md)
 * [What are the UA related APT messages?](./docs/explanations/apt_messages.md)
+* [How to interpret the security-status command](./docs/explanations/how_to_interpret_the_security_status_command.md)
+* [Why Trusty (14.04) is no longer supported?](./docs/explanations/why_trusty_is_no_longer_supported.md)
```

Comment:
Two options:
1. move the verb first to make it a question
```suggestion
* [Why is Trusty (14.04) no longer supported?](./docs/explanations/why_trusty_is_no_longer_supported.md)
```
2. remove the question mark
```suggestion
* [Why Trusty (14.04) is no longer supported](./docs/explanations/why_trusty_is_no_longer_supported.md)
```

Reply:
Going with the second one, since we don't have other questions in this section